src/drivers/intel/hda/hda-dma.c

```diff
@@ -571,10 +570,6 @@ static int hda_dma_release(struct dma_chan_data *channel)
 	 */
 	hda_chan->state |= HDA_STATE_RELEASE;
```

Comment:
is the comment and assignment still valid, if you don't have a work cb any longer?

Reply:
the assignment is valid because it helps avoid the FP increment during start for the release case. I'll remove the comment.

Reply:
what does 'FP' stand for @ranj063, I wasn't able to follow your answer.

Reply:
firmware pointer increment. we seem to increment it in the case of start and avoid it during release. I am not sure why though. I left it in as is for now as it doesnt really help avoid the xrun issue completely